pinot-core/src/test/java/org/apache/pinot/core/operator/transform/function/BaseTransformFunctionTest.java

```diff
@@ -580,9 +580,14 @@ protected void testTransformFunction(TransformFunction transformFunction, String
 
   protected void testTransformFunctionWithNull(TransformFunction transformFunction, String[] expectedValues,
       RoaringBitmap expectedNulls) {
+    testTransformFunctionWithNull(transformFunction, expectedValues, expectedNulls, false);
+  }
+
+  protected void testTransformFunctionWithNull(TransformFunction transformFunction, String[] expectedValues,
```

Comment:
We don't need to define this function because we can use the `testTransformFunction`.